packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -71,6 +71,34 @@ describe('Form-Input', () => {
       });
     });
 
+    describe('Enabled Placeholder', () => {
+      beforeEach(() => {
+        browser.url('/#/raw/tests/terra-form-input/form-input/placeholder-form-input');
+      });
+
+      Terra.should.beAccessible({ viewports });
+      Terra.should.matchScreenshot({ viewports });
+      Terra.should.themeCombinationOfCustomProperties({
+        testName: 'themed',
+        properties: {
+          '--terra-form-input-placeholder-color': 'red',
+          '--terra-form-input-placeholder-font-style': 'italic',
+        },
+      });
+    });
+
+    describe('Enabled Value', () => {
+      beforeEach(() => {
```

Comment:
same here

Reply:
Changed these as suggested. I also checked over the others I added. If there are any others outside of these that could use changing, let me know!